### ISSUE TITLE
multi-character code-points not supported

### ISSUE THREAD

**Issue by brianoncode** [eugene1g/font-blast#7]
I think there's an issue with 2-char code points. In `glyph-extractor`, `iconCode` is parsed using `charCodeAt(0)`, which doesn't support these higher up code points. See the second paragraph [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charCodeAt#Description) for explanation.

With a glyph line like this:
```xml
<glyph unicode="&#x1f308;" horiz-adv-x="2048" d="M1024 ..." />
```
I'm not seeing it hit the `if (iconCode.indexOf("&#") !== -1)` line. Instead the value of `iconCode` is the actual character 🌈.
It'll end up with `iconCode == '🌈'` instead of `iconCode == '1f308'` because `🌈.length === 1` is `false`. 

I think `glyph-extractor` should handle encoded `iconCode`s (ones with `&#`) differently from unencoded ones, which have the actual character in `iconCode`.

Something like this:
```js
// encoded
if (iconCode.indexOf("&#") !== -1) {
  iconCode = iconCode.replace("&#x", "");
} 
// unencoded
else {
  iconCode = iconCode.codePointAt(0).toString(16);
}
```

I can submit a PR for it.

**Comment by eugene1g** [comment]
Makes sense! Would love a PR on this :)

**Comment by eugene1g** [comment]
Released with `0.7.0`, thanks @brian123zx!
